ccs-scale-infra-shared/terraform/modules/infrastructure/cloudfront/functions/security-headers/index.js

```diff
@@ -12,8 +12,7 @@ exports.handler = (event, context, callback) => {
   }];
   headers['content-security-policy'] = [{
     key: 'Content-Security-Policy',
-    value: "default-src 'none'; img-src 'self' www.googletagmanager.com; script-src 'self' 'unsafe-inline' https://www.googletagmanager.com; font-src fonts.gstatic.com; style-src 'self' 'unsafe-inline' fonts.googleapis.com; object-src 'none'"
-
+    value: "default-src 'none'; img-src 'self' www.googletagmanager.com https://www.google-analytics.com; script-src 'self' 'unsafe-inline' https://www.googletagmanager.com https://tagmanager.google.com //www.google-analytics.com https://ssl.google-analytics.com; connect-src https://www.google-analytics.com; font-src fonts.gstatic.com; style-src 'self' 'unsafe-inline' fonts.googleapis.com https://tagmanager.google.com; object-src 'none'"
```

Comment:
Looks like missing `https:` on `//www.google-analytics.com`

Reply:
This is how it's listed in the documentation
![Screenshot 2021-05-07 at 15 15 03](https://user-images.githubusercontent.com/70591203/117470548-d6bdb980-af4e-11eb-8843-0940c2f7bac3.png)

Reply:
updated